pkgs/development/python-modules/python-jose-cryptodome/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, cryptography, ecdsa, future, pycryptodome_3_3, six }:
+
+buildPythonPackage rec {
+  pname = "python-jose-cryptodome";
+  version = "1.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0vl22411k394biizkjqg1ydclbf6fw5hxixzm1g7jlpzx4z6j9lj";
+  };
+
+  propagatedBuildInputs = [ cryptography ecdsa future pycryptodome_3_3 six ];
```

Comment:
Mhm this could be a big hassle if `python-jose-cryptodome` is mixed with something depending on `pycryptodome`. Then there are two different versions. which should be avoided.

Reply:
Maybe a newer version could be used here? https://github.com/mpdavis/python-jose/blob/master/setup.py#L58